src/modules/75-cf/pages/feature-flags/components/__tests__/FlagTableFilters.test.tsx

```diff
@@ -72,7 +112,26 @@ describe('FlagTableFilters', () => {
       expect(screen.getByText('cf.flagFilters.potentiallyStale')).toBeInTheDocument()
     })
 
-    fireEvent.click(screen.getByText('cf.flagFilters.permanent'))
-    await waitFor(() => expect(updateTableFilter).toBeCalled())
+    await act(async () => {
+      userEvent.click(screen.getByText('cf.flagFilters.permanent'))
+    })
```

Comment:
You probably don't need the `act` as `userEvent` does that for you
```suggestion
    userEvent.click(screen.getByText('cf.flagFilters.permanent'))
```